force-app/main/default/classes/OSF_AccountController.cls

```diff
@@ -0,0 +1,94 @@
+/**********************************************************************
+ * @Name         : OSF_AccountController
+ * @Description  : Class containing callable functions from LWC.
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 07 jul. 2022
+ * @Modification Log: 
+ ********************************************************************
+ * Version		Developer		Date			Description
+ *___________________________________________________________________
+ *
+ *********************************************************************/
+public with sharing class OSF_AccountController {
+/**********************************************************************
+ * @Name         : getOrdersAmountCurrentYear
+ * @Description  : Returns total of order totals since 1st January of current year 
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 07 jul. 2022
+ *
+ * @param String - string containing account id
+ * @return AggregateResult - result objects 
+ *********************************************************************/
+    @AuraEnabled(cacheable=true)
+    public static AggregateResult getOrdersAmountCurrentYear(String accountId) {
+        return [
+            SELECT SUM(TotalAmount) ordersAmountValue
+            FROM Order
+            WHERE accountId = :accountId AND EffectiveDate  = THIS_YEAR
+        ];
+    }
+
+/**********************************************************************
+ * @Name         : getOrdersAmountCurrentYearMinus5
+ * @Description  : Returns total of order totals since 1st January of current year - 5
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 07 jul. 2022
+ *
+ * @param String - string containing account id
+ * @return AggregateResult - result objects
+ *********************************************************************/
+    @AuraEnabled(cacheable=true)
+    public static AggregateResult getOrdersAmountCurrentYearMinus5(String accountId) {
```

Comment:
why you are calling different methods for each field - this will be one callout from LWC to server side. please create a single method to pull everything at once.

Reply:
Move the ordersAmmount for current year and for current year -5 into the wrapper class